### DISCUSSION TITLE
Share problem

### DISCUSSION THREAD

**Discussion by IWareQ** [community/community#5324] [Mobile]
add "Copy link" button![IMG_20210827_114857.jpg](https://user-images.githubusercontent.com/51165317/131092283-d55fad76-b0c8-431a-8e34-4977580ce03e.jpg)

**Comment by elisealix22** [comment]
Hi @IWareQ , thank you for reporting this! Which share link did you press to see this? Or is the copy URL option missing for all share links for you? Finally, do you mind sharing which device you are using so we can reproduce on our end?

**Reply by IWareQ** [reply]
Hello, i use Xiaomi Redmi Note 8. https://youtu.be/rfGaT0cadog
